raw_data/preprocessed/africa/tanzania_english.txt

```diff
@@ -0,0 +1,769 @@
+
+Cancellation of licence upon disapproval of Environmental Impact Assessment 
+(93) The Minister shall disapprove and recommend to the licensing authority that the project should not be licenced or, where the licence has been issued, be cancelled if: [a] the project or undertaking is likely to cause significant adverse impact on the environment; [b] there are no alternatives which can mitigate or remedy the significant likely harm to the environment; [c] the proponent has failed to abide to the mitigation measures stated in the Environmental Impact Statement or conditions issued by the Minister; or [d] there are compelling social, economic, health, cultural, or religious reasons which may or are likely to lead to irreversible impact on the society.
+
+Delegation of powers to approve Environmental Impact Assessment 
+(94) The Minister may, by an instrument published in the Gazette, delegate power of approval of Environmental Impact Statement to the Director of Environment, local government authorities or sector Ministries.
+
+Appeal against decision of the Minister on Environmental Impact Assessment 
+(95) Any person who is aggrieved by the decision of the Minister to approve or disapprove an Environmental Impact Statement Environmental Impact Statement may appeal to the Environmental Appeals Tribunal. 
+
+Maintenance of records of decisions on Environmental Impact Assessment 
+(96) [1] The Director-General shall maintain records of decisions on approvals or disapprovals of Environmental Impact Statement. [2] The record referred under subsection [1] shall be kept in a public registry and be accessible to the public upon payment of a prescribed fee. 
+
+Requirement of fresh Environmental Impact Assessment study 
+(97) The Council may, at any time after the issuance of an of Environmental Impact Assessment Certificate, require the holder of such certificate to conduct a fresh Environmental Impact Assessment mental study at his own cost and submit an Environmental Impact Statement within such time as the Council may prescribe if: [a] there is substantial change or modification in the project or undertaking or in the manner in which the project is being operated; [b] the project or undertaking poses environmental_ threat$ which could not be reasonably foreseen at the time of the study or review; or [c] in due course it is established that the information or data given by the proponent or undertaking in previous Environmental Impact Assessment process was inaccurate, false or intended to mislead the Council. 
```

Comment:
Please replace:
* `Assessment mental study` with `Assessment study`; and
* `environmental_ threat$` with `environmental threats`.